MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/model/Asset.java

```diff
@@ -53,4 +51,20 @@
 
     @ColumnInfo(name = "rt_rating")
     public String rottenTomatoesRating;
+    
+    public Asset(String id, AssetType type, String title, String poster, String banner, String plot,
```

Comment:
Some fields are optional here, such as imdbRating and rottenTomatoesRating.

Maybe consider using Builder? 
https://drive.google.com/file/d/1LRJ_aaMXu2EGamOc0bkoajtiWWi1NM13/view?usp=sharing
https://drive.google.com/file/d/1Jny5Nzpnhccxi9GEwz8ELl2FdHbvMycg/view?usp=sharing